package.json

```diff
@@ -59,6 +59,7 @@
   },
   "dependencies": {
     "@material/mwc-icon": "0.25.3",
+    "escape-string-regexp": "5.0.0",
```

Comment:
Adding another library would probably make it a bit slower, is there a way to do this without external libraries?